accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+        "packs": [
+            "Xamarin.iOS.Framework.14.0"
+        ]
+    },
+    "ios-build": {
+        "kind": "build",
+        "packs": [
+            "Xamarin.iOS.Framework"
+        ]
+    },
+    "ios": {
+        "description": "Create, build and run iOS apps",
+        "packs": [
+            "Xamarin.iOS.Framework",
+            "Xamarin.iOS.Templates",
+        ],
+        "extends": [ "ios-build" ]
+    }
+},
+"packs": {
+    "Xamarin.iOS.Templates": {
+        "version": "15.0.0",
+        "kind": "template"
+     },
+    "Xamarin.iOS.Framework": {
+        "version": "15.0.2",
+        "kind": "framework"
+     },
+    "Xamarin.iOS.Framework.14.0": {
+        "version": "14.2.3",
+        "kind": "framework",
+        "alias-to": {
+            "*": "Xamarin.iOS.Framework"
+        }
+    }
+}
+```
+
+The only purpose of these side-by-side versions is to make it so that servicing updates to an SDK series do not unexpectedly bring in API changes. They are not intended for long term use. **When a new SDK series is released, the versioned workloads and their packs should be removed from the manifest**.
+
+As demonstrated in this example, template packs should not be side-by-side versioned within an SDK band. Templates should always target the newest framework, automatically opting newly created projects into the newer of the side-by-side versions of the workload. If there is a developer scenario where specifying the framework is important then it should be exposed as a parameter on the templates rather than a separate template.
+
+## Expected SDK Resolution Behavior
+
+The MSBuild logic in the `WorkloadManifest.targets` that imports SDKs packs based on the `TargetPlatformVersion` MSBuild property should use it as a **lower bound** on the pack OS API version, and implement the following behaviors:
+
+**If multiple versions of the pack in the manifest comply with the version bound**, use the pack with the lowest version that complies with the platform API version bound. This means that when newer side by side packs become available in an SDK series, the project will not automatically use a newer version, but the MSBuild property can be used to opt in.
+
+**If any version of the pack in the manifest is higher than the version bound**, emit a warning telling the developer that a newer OS API version is available, and explaining how to update the MSBuild property. This will guide developers to keep the value current.
+
+**If no version of the pack in the manifest complies with the version bound**, emit an error telling the developer that their .NET SDK is too old.
+
+For example, if the SDK contains 14.0 and 15.0 APIs:
+
+- Project specifies 13.0 API: use 14.0 API, and emit a warning that newer API is being used
+- Project specifies 14.0 API: use 14.0 API, and emit a warning that newer API is available
+- Project specifies 15.0 API: use 15.0 API
+- Project specifies 16.0 API: emit an error that the SDK must be updated
```

Comment:
I think these rules are specific to the workload.  I don't think Windows is ever going to force you to a newer API version, and may not have side-by-side SDK versions.

Reply:
This pattern accounts for side by side versions. If they don't have side by side versions why would they not force you to a newer API version? It's literally the only option.